app/src/main/java/com/hover/stax/utils/Constants.kt

```diff
@@ -25,6 +25,8 @@ object Constants {
     const val PERMS_REQ_CODE = 700
 
     const val size55 = 55
```

Comment:
These should be in dimens.xml

Reply:
The previously added constant is no more in use.

We could probably do a constant cleanup as a chore story.

Reply:
@davkutalek , thanks for the reviews. Changes effected.

One UI note is unclear: The balance/amount line above the cost line is still missing.
Do you mean this below?

![TDscreenshot](https://user-images.githubusercontent.com/21104847/163500875-95b1e3eb-f613-4c83-9aa9-fe47afd46c47.png)

Reply:
@androidmaven yep. The recipient account and to lines should also be below the gap